src/components/JdsPagination/Pagination.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div
+    :class="{
+      'jds-pagination font-sans-1': true,
+      'jds-pagination--disabled': disabled,
+    }"
+  >
+    <div class="jds-pagination__wrapper">
+      <!-- left section -->
+      <div class="jds-pagination__page-control--left">
+        <div class="jds-pagination__page-control__select">
+          <span>
+            Tampilkan
+          </span>
+          <select :value="mItemsPerPage" @change="onItemsPerPageChange" :disabled="disabled">
+            <option :value="item" v-for="item in itemsPerPageOptions" :key="item">
+              {{ item }}
+            </option>
+          </select>
+          <span>
+            item
+          </span>
+          <i class="jds-pagination__divider" />
+          <span>dari total <strong>{{ totalRows }}</strong></span>
+          <i class="jds-pagination__divider" />
+        </div>
+      </div>
+
+      <!-- rigth section -->
+      <div class="jds-pagination__page-control--right">
+        <button
+          class="jds-pagination__navigation-button"
+          :disabled="disabled"
+          @click="onPreviousPage"
+        >
+          <jds-icon name="chevron-left" size="sm" />
+        </button>
+        <i class="jds-pagination__divider" />
+        <div class="jds-pagination__page-control__select">
+          <span>Halaman</span>
+          <select :value="mCurrentPage" @change="onPageChange" :disabled="disabled">
+            <option :value="page" v-for="page in generatedPageNumbers" :key="page">
+              {{ page }}
+            </option>
+          </select>
+          <span>dari <strong>{{ pages }}</strong></span>
+        </div>
+        <i class="jds-pagination__divider" />
+        <button class="jds-pagination__navigation-button" :disabled="disabled" @click="onNextPage">
+          <jds-icon name="chevron-right" size="sm" />
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import JdsIcon from '../JdsIcon'
+import localCopy from '../../mixins/local-copy'
+
+export default {
+  name: "jds-pagination",
+  components: {
+    JdsIcon
+  },
+  mixins: [
+    localCopy('currentPage', 'mCurrentPage'),
+    localCopy('itemsPerPage', 'mItemsPerPage')
+  ],
+  data() {
+    return {
+      mCurrentPage: null,
+      mItemsPerPage: null,
+    }
+  },
+  props: {
+
+    /**
+     * Bound model.
+     * @name currentPage
+     * @model
+     */
+    currentPage: {
+      type: [Number, String],
+      default: 1
+    },
+
+     /**
+     * Bound model.
+     * @name itemsPerPage
+     * @model
+     */
```

Comment:
`@model` should only be used once, since `@model` reflects v-model attribute.

Reply:
thank you @adrianpdm , will fix it ASAP!

Reply:
fixed on this commit @adrianpdm [bf05a73](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/43/commits/bf05a739a39459f3f0d5a172012ce7f5999b1800)